### DISCUSSION TITLE
Downloading Older Versions of Artisan?

### DISCUSSION THREAD

**Discussion by ca-zubair-hassam** [artisan-roaster-scope/artisan#1109] [Q&A]
Hey fellow roasters.

I wanted to know if it is possible to download and run older versions of Artisan for windows?

**Comment by poundy** [comment]
Well possibly. But why? There's nothing the current cersion doesn't do, as far as I'm aware. And if there is it's possibly something worth discussing rather than just reverting to a software version (and components) that are outdated and no longer supported.
